tests/test_torch_agent.py

```diff
@@ -49,6 +49,9 @@ def __getitem__(self, key):
             self.idx += 1
             return self.idx
 
+    def get(self, key):
```

Comment:
This shouldn't be here, DictAgent doesn't have a `get`, so the mock dictagent shouldn't have one.

Reply:
fixed